examples/project_template_webpack_vue/main.cpp

```diff
@@ -4,6 +4,7 @@
 #include "logger.h"
 #include <string>
 #include <vector>
+#include <experimental/array>
```

Comment:
There shouldn't be any nneed for the `experimental` qualifier. gcc 4.9 forwards support `std::array`.